price-pusher/tests/unit/core/test_listener.py

```diff
@@ -66,17 +66,20 @@ async def test_fetch_all_oracle_prices(price_listener, mock_request_handler, moc
         pair_id="BTC/USD",
         price=10000,
         timestamp=1234567890,
-        source="source_1",
-        publisher="publisher_1",
+        source="SOURCE_1",
```

Comment:
create constants for source name and publisher name

Reply:
done